aws/cloudformation/cloud_formation_stack.yml.erb

```diff
@@ -432,6 +432,9 @@ Resources:
       Tags:
         - {Key: Name, Value: <%=frontends ? "#{environment}-daemon" : "!Ref 'AWS::StackName'"%>}
       BlockDeviceMappings:
+        # BlockDeviceMappings cannot be modified after instance creation
```

Comment:
Should we be more explicit and state in the comment that modifying the Volume size will delete the daemon instance and recreate it?